backend/.env.example

```diff
@@ -11,11 +11,9 @@ CLOUDINARY_FOLDER=
 SENDGRID_API_KEY=
 SENDGRID_FROM_EMAIL=
 SENDGRID_FROM_NAME=
+SENDGRID_ACCEPTED_TEMPLATE=

```

Comment:
I removed these two in a previous commit to dev, you should remove them from your local .env so that they don't get copied over to .env.example.